cluster/cluster.yaml

```diff
@@ -124,6 +124,12 @@ Resources:
           FromPort: 8085
           IpProtocol: tcp
           ToPort: 8085
+{{- if eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam "true" }}
```

Comment:
This should be dropped. I think it's from when the webhook as deployed as it's own sidecar.

Reply:
I totally missed that. Removed the related stuff.